app/controllers/orders_controller.rb

```diff
@@ -37,4 +32,18 @@ def create
   def order_params
     params.require(:order).permit(:employee_id, :reward_id, :reward_snapshot)
   end
+
+  def address_params
+    params.require(:address).permit(:street, :postcode, :city) if params[:address]
+  end
+
+  def order_form_params
+    order_params.merge(address_params)
+  end
+
+  def which_order_form
+    return OrderPostForm if @reward.delivery_method == 'post_delivery'
```

Comment:
@reward.delivery_method == 'post_delivery' ? OrderPostForm : OrderOnlineForm

you can also use delivery_method enum here

Reply:
Thanks! Done 👍  My bad :D